.github/workflows/stateful-build-wheels.yml

```diff
@@ -0,0 +1,149 @@
+name: Jina v-raft Workflow
```

Comment:
this is supposed to be a temporary file?

Reply:
yes, before merging jina-v-raft to master, we don't rely on `stateful` tag anymore and incorporate the wheels building step in the regular release workflow